frontend/src/tabs/zoom.pug

```diff
@@ -1,8 +1,13 @@
 #zoom
+  span.large-font Commits Panel
   .zoom__title
     .zoom_title--group.large-font
-      span(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
-      span(v-else) {{ info.user.repoName }}
+       span(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
+       a(
+       v-bind:href="info.location", target="_blank",
+       v-bind:title="'Click to open the repo'"
+       )
+         span(v-if="info.filterGroupSelection !== 'groupByAuthors' ") {{ info.user.repoName }}
```

Comment:
`v-if` should be put inside `<a>`, otherwise, there may be a link inside HTML with nothing inside, which is a bad practice. Do note the indentation as well.
![image](https://user-images.githubusercontent.com/35621726/70388925-51e60d80-19f3-11ea-8168-5594f08a12ea.png)
Suggestion: use `v-else` for simplesity:
```
span(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
a(
  v-else,
  v-bind:href="info.location", target="_blank",
  v-bind:title="'Click to open the repo'"
)
  span {{ info.user.repoName }}
```

Reply:
Would you prefer if I keep the link or remove it?

Reply:
You may just use my edited version.

Reply:
> **EDIT 1:** Is it that the link will be created irrespective of the selected choice and that is a bad practice? 

Correct. It is not a good practice because it is leaving extra information that is not used in the HTML. Although it is not reflected in the website, others can find the link easily in the raw HTML.
 
> Though it kind of makes sense as an empty link if found can be used for attacks

Correct. Of course, this link you leave here will no cause any security issue. But what if you are building a secure website next time? As a high-standard SE repo, it is nice if we can always keep the high standard in mind.

Reply:
Got it. Thanks for explaining.